datafusion/sql/src/unparser/expr.rs

```diff
@@ -561,6 +636,31 @@ mod tests {
                 }),
                 r#"CAST("a" AS INTEGER UNSIGNED)"#,
             ),
+            (
+                Expr::InList(InList {
+                    expr: Box::new(col("a")),
+                    list: vec![lit(1), lit(2), lit(3)],
+                    negated: false,
+                }),
+                r#""a" IN (1, 2, 3)"#,
+            ),
+            (
+                Expr::InList(InList {
+                    expr: Box::new(col("a")),
+                    list: vec![lit(1), lit(2), lit(3)],
+                    negated: true,
+                }),
+                r#""a" NOT IN (1, 2, 3)"#,
+            ),
+            (
+                Expr::ScalarFunction(ScalarFunction {
+                    func_def: ScalarFunctionDefinition::UDF(Arc::new(
+                        ScalarUDF::new_from_impl(DummyUDF::new()),
+                    )),
+                    args: vec![col("a"), col("b")],
+                }),
+                r#"dummy_udf("a", "b")"#,
+            ),
```

Comment:
I think for `ScalarFunction`, it's better to add test cases for `UDF` and `BuiltIn`.
But since the function port work is going to be finished, I doubt whether it's necessary to do that. :)

Reply:
Since there is no code specific to `BuiltInScalarFunction` in the unparser code, I agree there is no reason to add a specific test here